src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
```

Comment:
The config property you are adding is called `meta` but the one you accessing here is called `ethpm`.

Did you intend to add `meta` to the `eth-pm` plugin config rather than the global `ConfigManager`?
If so, that would be in the `ape_pm` directory, you would create a class that implements the `PluginConfig` base class and register that as a config in the `__init__.py` file of that directory.

Reply:
Agree, rename the property to `meta`, I think we should standardize on this terminology here (abandoning `ethpm`)

Reply:
resolved in [67ccc3a](https://github.com/ApeWorX/ape/pull/802/commits/67ccc3a11db191c3a9eaf94acf94b25e3a561b5b)